client/js/compute.js

```diff
@@ -684,6 +684,48 @@ const compute_ops = [
     sample: 'var a = randRange ${x} ${y} ${z}',
     call: function(v, x, y, z) { return v = Math.round(Math.floor((Math.random() * (y - x) / (z || 1))) * (z || 1) + x) },
     hash: '4758cba8f404cfe3ef0a300cebf3938a'
+  },
+  {
+    name: 'colorContrast',
+    desc: 'converts color in any format that the browser can interpret to another color in the same hue but with different luminance',
+    sample: 'var a = colorContrast ${x} ${y}',
```

Comment:
This doesn't explain which parameter does what and the format of the second parameter. Not sure how that compares to existing functions.